shared/actions/profile/proofs.js

```diff
@@ -38,6 +41,18 @@ const checkProof = (state, action) => {
     })
 }
 
+const recheckProof = (state, action) =>
+  flags.identify3 &&
+  RPCTypes.proveCheckProofRpcPromise({sigID: action.payload.sigID}, Constants.waitingKey).then(() =>
+    Tracker2Gen.createLoad({
```

Comment:
@chrisnojima this seems heavy handed, does it seem right to you? I was looking for something that triggers `identify3UpdateRow`

Reply:
lets keep it simple, it should be fast as everything should be cached